src/App.tsx

```diff
@@ -1,120 +1,126 @@
-import { ChangeEvent, FormEvent, useState } from 'react'
-import {Header} from './components/Header'
-import { Search } from './components/Search'
-import imgClipboard from './assets/Clipboard.svg'
-import styles from './App.module.css'
-import './global.css'
-import { TaskViewSimple } from './features/task/components/TaskViewSimple'
-import { Task, TaskStatus } from './features/task/types/Task'
+import { ChangeEvent, FormEvent, useState } from "react";
+
+import "./global.css";
+import { Search } from "./components/Search";
+import { Task, TaskStatus } from "./features/task/types/Task";
+import { TaskView } from "./features/task/components/TaskView";
+import { Header } from "./components/Header";
+import imgClipboard from "./assets/Clipboard.svg";
+import styles from "./App.module.css";
 
 function App() {
-  
-  const [currentTask, setCurrentTask] = useState('')
+  const [numberTask, setNumberTask] = useState<number>(1);
+  const [currentTask, setCurrentTask] = useState("");
+  const [focusTask, setFocusTask] = useState<number>(0);
   const [tasks, setTasks] = useState<Task[]>([]);
 
-  //Function response to the add the task to list of tasks
-  function onIncludeTask(event : FormEvent) {
+  function handleChangeFocus(currentTaskId: number) {
+    setFocusTask(currentTaskId);
+  }
 
-    event.preventDefault()
+  //Function response to the add the task to list of tasks
+  function onIncludeTask(event: FormEvent) {
+    event.preventDefault();
 
     let currentTaskComplete: Task = {
-        title: currentTask,
-        status: TaskStatus.UNCOMPLETED
-    }
+      id: numberTask,
+      title: currentTask,
+      status: TaskStatus.PENDING,
+      until: "15/01/2024 21:00",
```

Comment:
Não sei sobre regra de negócio, mas não seria interessante você definir esse until como um valor dinâmico?

Reply:
Ainda to desenvolvendo essa parte, tenho uma task pra isso, deixei mockado mais pra ter um dado base mesmo.